python/visualization.py

```diff
@@ -447,6 +450,31 @@ def sort_points(xy):
     return ax
 
 
+def _add_colorbar(
+    ax: Axes,
+    cmap: str,
+    vmin: float,
+    vmax: float,
+    label: str,
+    clip_values: bool = False,
+) -> None:
+    """Add a colorbar to the parent Figure of 'ax' by creating an additional Axes."""
+    import matplotlib as mpl
+    from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+    # Create a map between field/eps values and colors in the colormap.
+    # Note: cm.get_cmap() is deprecated for matplotlib>=3.6, use mpl.colormaps[cmap] instead if necessary.
+    sm = mpl.cm.ScalarMappable(
+        norm=mpl.colors.Normalize(vmin, vmax, clip_values), cmap=mpl.cm.get_cmap(cmap)
+    )
+    plt.colorbar(
+        mappable=sm,
+        cax=make_axes_locatable(ax).append_axes(position="right", size="5%", pad=0.05),
```

Comment:
Any reason why the `pad` parameter is chosen as an absolute quantity rather than a percentage (e.g., `2%`) similar to `size`?

Reply:
Good catch, I agree that using the relative quantity is better here.